VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.7.0
+1.8.0
```

Comment:
You might want to check with inference team, as bumping it up may affect patch releases. A less intrusive change is to parse and bump up minor version inside setup.py when building nightly.